source/jormungandr/jormungandr/utils.py

```diff
@@ -494,6 +494,30 @@ def get_pt_object_from_json(dict_pt_object, instance):
     return pt_object
 
 
+def transform_entrypoint(dict_pt_object, uri):
+    if not dict_pt_object:
+        return None
+    if MAP_STRING_PTOBJECT_TYPE.get(dict_pt_object.get("embedded_type")) != type_pb2.ADDRESS:
+        return dict_pt_object
```

Comment:
I'm wondering if you should have a second sub-function here. The goal of a sub-function is for documentation. Since the sub-function also has a name, it helps to understand a sub-part of the code.
```suggestion
    if MAP_STRING_PTOBJECT_TYPE.get(dict_pt_object.get("embedded_type")) == type_pb2.ADDRESS:
        replace_address_with_custom_poi(dict_pt_object, uri)
    else:
        return dict_pt_object
```

All the code below would go in the new function.

Reply:
Done